notebooks/correlation_vs_overlap.ipynb

```diff
@@ -0,0 +1,1735 @@
+{
```

Comment:
<p>I think this would work:</p><pre class="ql-syntax" spellcheck="false">fields1, fields2 = zip(*pairs)
</pre><p>But if what you have seems clearer to you I'm fine with that - <code>zip</code>'s behavior is always a bit confusing. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/indices/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>